### ISSUE TITLE
`license` and `attribution`

### ISSUE THREAD

**Issue by larsbuntemeyer** [WCRP-CORDEX/cordex-cmip6-cv#3]
Probably required for dowscaling CC BY 4.0 data (CMIP6, ERA5)

```json
{
    "attribution": "CORDEX model data is based on downscaling <driving_source_id> driving data produced by <driving_institution_id> using the regional model <source_id>. The driving data is licensed under a Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses)."
}
```

**Comment by larsbuntemeyer** [comment]
Just for my convenience, here are the two attributions:

* `ECMWF-ERA5`: `attribution = "This service is based on data and products of the European Centre for Medium-Range Weather Forecasts (ECMWF). The driving model ECMWF-ERA5/ERA5.1 data is published under a Creative Commons Attribution 4.0 International (CC BY 4.0). https://creativecommons.org/licenses/by/4.0/"` according to [ECMWF - Use of data](https://apps.ecmwf.int/datasets/licences/general/).
* `CMIP6 GCM`: `attribution = CMIP6 model data produced by <Specific Modeling Centre Name> is licensed under a Creative Commons Attribution 4.0 International License (CC BY 4.0; https://creativecommons.org/licenses/). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file). The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.` according to [CMIP6 terms of use](https://pcmdi.llnl.gov/CMIP6/TermsOfUse).

**Comment by larsbuntemeyer** [comment]
license is now fixed as a path to the ToU: https://cordex.org/data-access/cordex-cmip6-data/cordex-cmip6-terms-of-use

attribution is optional and should be done during publications
